config.example.wikimedia.yaml

```diff
@@ -184,6 +196,97 @@ templates:
 #              service: 
 #                uri: http://revscore.wikimedia.org/{domain}/{title}/{revision}
 
+  global-sys: &gb/sys/1.0.0
+    info:
+      title: global domain sys API module
+      version: 1.0.0
+    paths:
+      /{module:table}: *wp/sys/table
+      /{module:key_value}: *wp/sys/key_value
+      /{module:post_data}: *wp/sys/post_data
+      /{module:mathoid}/v1/request/{format}:
```

Comment:
In https://github.com/wikimedia/restbase/pull/336 and https://github.com/wikimedia/restbase/pull/360 we've added a client-ip check for endpoints that are using saving to `post_data` module. 

So, this endpoint should be protected with a `header_match` security stanza, but I'm not sure how(if) it's going to work with it (who is POSTing here?)